src/GIRUtils.php

```diff
@@ -2,40 +2,67 @@
 
 namespace Drupal\os2forms_egir;
 
+use GuzzleHttp\Client;
+
 use GuzzleHttp\Exception\BadResponseException;
 
 /**
  * Utilities for GIR communication & EGIR form data.
  */
 class GIRUtils {
 
```

Comment:
Vi har vel egentlig to former for utils, hvis det skal være rigtigt - utils til at snakke med GIR API'et (som skal bruge en HTTP-client), og utils til at hive data ud af Drupal. Spørgsmålet er, om vi skal have splittet det op, så vi ikke slæber rundt på en HTTP-client, vi ikke skal bruge.

Det er bare en tanke - jeg synes det er en fin omstrukturering uanset.

Reply:
Helt rigtigt! Og det kunne potentielt ganske rigtigt være en god idé med en sådan opsplitning, men så vil jeg gerne vente og se, hvor meget utils går hen og vokser først.

Reply:
Helt enig, lad os bare vente.